WordPress/build.gradle

```diff
@@ -92,9 +92,9 @@ android {
             dimension "buildType"
             // Only set the release version if one isn't provided
             if (!project.hasProperty("versionName")) {
-                versionName "15.9-rc-2"
+                versionName "15.9"
```

Comment:
Side note: do we already have a Peril rule to check that we don't commit `rc` versions on release PRs? (Just wondering if we're supposed to double-check that manually on release PRs or if Peril already takes care of it)

Reply:
We don't have a Peril rule for this. We also merge back `rc`s after every beta so that changes land there quickly. We can think about how to create a Peril rule.